docs/sql-guide/functions/function-datetimeadd.md

```diff
@@ -0,0 +1,105 @@
+---
+title: DATETIMEADD()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# DATETIMEADD() function
+
+The `DATETIMEADD()` function adds a time duration specified using time unit and integer value to a date-time and then returns the results.
+
+## Syntax
+
+```
+DATETIMEADD(timeunit, int_expr, date-time)
+```
+
+## Arguments
+
+| Argument | Data type | Description | Required | Further information |
+|---|---|---|---|---|
+| timeunit | string | String value that specifies the time unit for the duration to be added to the target date-time. | Yes | [Time units](#additional-information).|
```

Comment:
The link under "further information" for timestamp should be the anchor for "timeunit" not the parent heading